rotkehlchen/api/rest.py

```diff
@@ -1555,6 +1557,94 @@ def process_history(
         result_dict = _wrap_in_result(result=result, message=msg)
         return api_response(result_dict, status_code=status_code)
 
+    def _get_history_debug(
+        self,
+        from_timestamp: Timestamp,
+        to_timestamp: Timestamp,
+    ) -> Dict[str, Any]:
+        """This method creates all history events for a timestamp range.
```

Comment:
I would that it searches or smth else rather than `creates`

Reply:
+1. Use exports.